sos/plugins/openstack_ironic.py

```diff
@@ -50,13 +50,16 @@ def setup(self):
         vars_any = [p in os.environ for p in [
                     'OS_TENANT_NAME', 'OS_PROJECT_NAME']]
 
-        if not (all(vars_all) and any(vars_any)):
+        self.osc_available = all(vars_all) and any(vars_any)
+
+        if not self.osc_available:
```

Comment:
Commits aren't expensive - I know it's tempting to cram internal refactoring in with other changes, but including this all in one commit makes your commit message a bit misleading. There's no hint of these changes in the four bullets that are mentioned.

Reply:
Will do

Reply:
This comment was fixed, but I don't know how to mark is as such..